src/lib/badges/Badge.svelte

```diff
@@ -1,26 +1,24 @@
 <script lang="ts">
   import classNames from 'classnames';
-  import CloseButton from '../utils/CloseButton.svelte';
```

Comment:
This CloseButton component's style is hard to modify, so I directly make changes on a element (see beow).

Reply:
`CloseButton` is supposed to be re-used among different components so I don't like hardcoding it in the badge.